### DISCUSSION TITLE
Output vector of the form [x(t0), x[t1], x[t2]]?

### DISCUSSION THREAD

**Discussion by raihaan123** [lululxvi/deepxde#896] [Q&A]
Hi everyone,

I'm Raihaan - I'm relatively new to DeepXDE but have played with it a little and am impressed at how neatly it encapsulates PINN training! Very briefly I am a newcomer to the PINN landscape and am presently researching methods to improve their trained accuracy for simple ODE problems (see [here](https://github.com/raihaan123/UROP-Notes/blob/main/src/runner.py) for the Van der Pol oscillator). I've thrown together a rudimentary W&B pipeline to automate experiments in the process which has yielded some interesting and fun results.

For the next phase of my research, I want the PINN to evaluate the function for a predefined patch surrounding and including the requested time t (or x in code sometimes?) It is of course easy enough to add dimensions to the output layer of `dd.nn.FNN`, but is there a simple way to augment the loss function, perhaps like [ode_t-1, ode_t, ode_t+1]?

I haven't likely explained this properly, please do shoot me any further questions you may have!

Many thanks :))

**Comment by lululxvi** [comment]
I don't fully understand what you try to do.
